### DISCUSSION TITLE
Installation issue

### DISCUSSION THREAD

**Discussion by minw1012** [TDAmeritrade/stumpy#687] [General]
![Screen Shot 2022-09-29 at 11 31 40 AM](https://user-images.githubusercontent.com/14137107/193115126-988474cc-d5a5-40c7-a98e-50ae0868d636.png)

Anyone can help me fix this issue?

**Comment by seanlaw** [comment]
Can you try updating `numpy`, `scipy`, and `numba` to the latest versions? It seems that your Python environment is not compatible with the installed version of `stumpy`.

What version of each of these are you currently running?

**Comment by minw1012** [comment]
Thanks for your time.

numpy 1.23.3
scipy  1.9.1
numba  0.56.2
Python 3.9.13
stumpy  1.11.1

**Reply by seanlaw** [reply]
According to your error, this looks like a `numba` issue. Can you try downgrading setuptools to be less than version 60?

**Reply by minw1012** [reply]
I changed to setuptools 59.8.0
It still didn't work.
Thanks

**Reply by minw1012** [reply]
It works in Python 3.8 instead of python 3.9.

**Reply by seanlaw** [reply]
Glad you were able to figure it out @minw1012. STUMPY is compatible and tested with Python 3.6+ on all major platforms and should work for Python 3.9 and Python 3.10.
